mongo/client_test.go

```diff
@@ -179,6 +179,26 @@ func TestClient(t *testing.T) {
 		client := setupClient(options.Client().SetReadConcern(rc))
 		assert.Equal(t, rc, client.readConcern, "expected read concern %v, got %v", rc, client.readConcern)
 	})
+	t.Run("min pool size", func(t *testing.T) {
+		testCases := []struct {
+			name string
+			opts *options.ClientOptions
+			err  error
+		}{
+			{"minPoolSize < default maxPoolSize", options.Client().ApplyURI("mongodb://localhost:27017/?minPoolSize=64"), nil},
```

Comment:
Optional: Consider using keyed fields and breaking out the test case structs onto multiple lines to make them easier to read.

E.g.
```go
{
	name: "minPoolSize < default maxPoolSize",
	opts: options.Client().ApplyURI("mongodb://localhost:27017/?minPoolSize=64"),
	err: nil,
},
```